test/algorithms/regressors/test_neural_network_regressor.py

```diff
@@ -122,6 +110,24 @@ def callback(objective_weights, objective_value):
             callback=callback,
         )
 
+        return regressor, regression_opflow_qnn, ansatz
+
+    @idata(itertools.product(OPTIMIZERS, QUANTUM_INSTANCES, CALLBACKS))
+    @unpack
+    def test_regressor_with_opflow_qnn(self, opt, q_i, cb_flag):
+        """Test Neural Network Regressor with Opflow QNN (Two Layer QNN)."""
+        if cb_flag is True:
```

Comment:
Arguably this would be better as simply `if cb_flag` since we really only need a flag that evaluates to True - if I set CALLBACKS to [1, 0] even though they evaluate to True, False, since it does not have the actual True object (singleton) there it would fail to ever test the callback.